pom.xml

```diff
@@ -348,14 +350,47 @@ under the License.
         </profile>
 
         <profile>
-            <id>scala-2.13</id>
+            <id>spark3</id>
+            <modules>
+                <module>paimon-spark/paimon-spark-3.x-common</module>
+                <module>paimon-spark/paimon-spark-3.5</module>
+                <module>paimon-spark/paimon-spark-3.4</module>
+                <module>paimon-spark/paimon-spark-3.3</module>
+                <module>paimon-spark/paimon-spark-3.2</module>
+            </modules>
             <properties>
+                <scala.binary.version>2.12</scala.binary.version>
+                <scala.version>${scala212.version}</scala.version>
+                <paimon-spark-common.spark.version>3.5.3</paimon-spark-common.spark.version>
+                <paimon-spark-x.x.common>paimon-spark-3.x-common</paimon-spark-x.x.common>
+                <test.spark.main.version>3.3</test.spark.main.version>
```

Comment:
`test.spark.version` ?
and why just test spark 3.3 ?

Reply:
because paimon use https://github.com/big-data-europe/docker-spark to run e2e test now, and it only supports up to 3.3.0 now, add a todo here